File: rust_internal_api_system/src/grpc_key_string_maps/account_auth_service.rs
pub struct HelloRequest {
}
                    
impl HelloRequest {
    pub const name: &str = "name";
}
                    


pub struct HelloReply {
}
                    
impl HelloReply {
    pub const message: &str = "message";
}
                    


pub struct RegisterRequest {
}
                    
impl RegisterRequest {
    pub const email: &str = "email";
}
                    


pub struct RegisterReply {
}
                    
impl RegisterReply {
    pub const result: &str = "result";
    pub const error: &str = "error";
}
                    


pub struct RegisterConfirmRequest {
}
                    
impl RegisterConfirmRequest {
    pub const email: &str = "email";
    pub const random_string: &str = "random_string";
}
                    


pub struct RegisterConfirmReply {
}
                    
impl RegisterConfirmReply {
    pub const jwt: &str = "jwt";
    pub const error: &str = "error";
}
                    


pub struct IsJwtOkRequest {
}
                    
impl IsJwtOkRequest {
    pub const jwt: &str = "jwt";
}
                    


pub struct IsJwtOkReply {
}
                    
impl IsJwtOkReply {
    pub const email: &str = "email";
    pub const error: &str = "error";
}

File: python_chat_with_friends_system/src/grpc_key_string_maps/account_auth_service.py

from typing import List

class HelloRequest:
    name: str = "name"
    __property_list__: List[str] = ["name"]

class HelloReply:
    message: str = "message"
    __property_list__: List[str] = ["message"]

class RegisterRequest:
    email: str = "email"
    __property_list__: List[str] = ["email"]

class RegisterReply:
    result: str = "result"
    error: str = "error"
    __property_list__: List[str] = ["result", "error"]

class RegisterConfirmRequest:
    email: str = "email"
    random_string: str = "random_string"
    __property_list__: List[str] = ["email", "random_string"]

class RegisterConfirmReply:
    jwt: str = "jwt"
    error: str = "error"
    __property_list__: List[str] = ["jwt", "error"]

class IsJwtOkRequest:
    jwt: str = "jwt"
    __property_list__: List[str] = ["jwt"]

class IsJwtOkReply:
    email: str = "email"
    error: str = "error"
    __property_list__: List[str] = ["email", "error"]

class IsOnlineRequest:
    email: str = "email"
    __property_list__: List[str] = ["email"]

class IsOnlineResponse:
    error: str = "error"
    online: str = "online"
    __property_list__: List[str] = ["error", "online"]